src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -69,7 +72,19 @@ public final class InlineConfigParser {
      * These are matched by
      * {@link InlineConfigParser#VIOLATION_WITH_EXPLANATION_PATTERN}.
      * </p>
-     */
+     * <p>
+     * Multiple violations for line. Violations are Y lines above, messages are X lines below.
+     * {@code
+     *   // X violations Y lines above:
+     *   //                            'violation message1'
+     *   //                            'violation messageX'
+     * }
+     *
+     * These are matched by
+     * {@link InlineConfigParser#MULTIPLE_VIOLATIONS_SOME_LINES_ABOVE_PATTERN}
+     * and {@link InlineConfigParser#VIOLATION_MESSAGE_PATTERN}
+     * </p>
+     * */
```

Comment:
Why is this javadoc here?

Reply:
All others are here , for some weird reason, I just extended.
I think this should be moved to class to show possible cases of usage.

Reply:
Shouldn’t this be on MULTIPLE_VIOLATIONS_SOME_LINES_ABOVE_PATTERN?

Reply:
moved there.